docs/man/man3/nxt_unit_done.3

```diff
@@ -0,0 +1,54 @@
+.\" (C) 2023, NGINX, Inc.
+.\"
+.TH nxt_unit_done 3 (date) "NGINX Unit (unreleased)"
+.SH Name
+nxt_unit_done \- destroy context object in Unit app
+.SH Library
+NGINX Unit library
+.RI ( libunit ", " -lunit )
+.SH Synopsis
+.nf
+.B #include <nxt_unit.h>
+.PP
+.BI "void nxt_unit_done(nxt_unit_ctx_t *" ctx );
+.fi
+.SH Arguments
+.TP
+.I ctx
+Context object to be destroyed.
+It should have been created with
+.MR nxt_unit_init 3
+or
+.MR nxt_unit_ctx_alloc 3 .
+.SH Description
+.MR nxt_unit_done 3
+destroys a context object in a Unit application.
```

Comment:
Might be worth noting that this must happen before the parent context is destroyed (at least as far as I understand from looking at the code).

Reply:
Yep, worth mentioning.